src/index.html

```diff
@@ -1,14 +1,73 @@
-<!doctype html>
+ <!doctype html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="#Home">
+        <img src="images/logo.png" alt="Home">
+      </a>
+      <nav>
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="Apple">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
```

Comment:
```suggestion
      <a class="logo" href="#Home">
        <img src="images/logo.png" alt="Home">
      </a>
      <nav>
        <ul class="nav__list">
          <li class="nav__item">
            <a class="nav__link is-active" href="Apple">
              Apple
            </a>
          </li>
          
          <li class="nav__item">
```
fix code style all over the page